Cargo.toml

```diff
@@ -18,8 +18,14 @@ anyhow = "1.0.71"
 clap = { version = "4.3.0", optional = true, features = ["unicode", "derive"] }
 env_logger = "0.10.0"
 log = "0.4.17"
-nu-parser = "0.81.0"
-nu-protocol = "0.81.0"
+nu-cli = { path = "../nushell/crates/nu-cli", version = "0.81.1", features = [
+    "plugin",
+] }
+nu-cmd-lang = { path = "../nushell/crates/nu-cmd-lang", version = "0.81.1" }
+nu-command = { path = "../nushell/crates/nu-command", version = "0.81.1" }
+nu-explore = { path = "../nushell/crates/nu-explore", version = "0.81.1" }
+nu-parser = { path = "../nushell/crates/nu-parser", version = "0.81.1" }
+nu-protocol = { path = "../nushell/crates/nu-protocol", version = "0.81.1" }
```

Comment:
this does not look good at all to me, does it? :open_mouth:

Reply:
Right, it is bad, but `cargo install` gets me the 0.81 version 
https://crates.io/crates/nu

Reply:
Is there another way to use 0.81.1? I didn't think there was.

Reply:
maybe we should just use `0.xx.0` then :thinking:

Reply:
It fails to read the engine. I don't remember what was, but there is a new function or public feature of 0.81.1 that is not available in 0.81.0. That's why I needed to install the patched version.
Are there any plans to publish the version of nushell 0.81.1?

Reply:
@AucaCoyan
you should be able to use `0.82.0` from yesterday :wink:

Reply:
https://crates.io/crates/nu

Reply:
Deps updated!

Reply:
looks better :yum: